### DISCUSSION TITLE
40

### DISCUSSION THREAD

**Discussion by giscus** [car-ffeine/car-ffeine.github.io#21] [Announcements]
# 40

안녕하세요? 센트와 가브리엘 입니다.

https://car-ffeine.github.io/40

**Comment by yogjin** [comment]
감명받았어요.. 프로젝트에 진심이 느껴집니다.
글도 너무 재밌게 잘 읽었습니다👍

**Reply by gabrielyoon7** [reply]
감사합니다.

**Comment by drunkenhw** [comment]
여기 팀은 천재들만 있나보네요

**Reply by gabrielyoon7** [reply]
저는 아닙니다

**Comment by Gilpop8663** [comment]
팀원이랑 전기차 충전소를 돌며 재밌는 추억 쌓고 계시네요 👍👍👍 

글 잘 보았습니당

**Reply by gabrielyoon7** [reply]
감사합니다.
